### ISSUE TITLE
Active elements use invalid and duplicate HTML id attributes

### ISSUE THREAD

**Issue by Mankro** [apluslms/a-plus#823]
@Mankro The problem also exists with active element exercises, whose forms are constructed by Javascript code instead of mooc-grader.

In fact, the problem is even worse because the label's `for` and the input's `id` attributes don't match. For example, in the Exponential distribution exercise, the label's `for` attribute is `exponentlambdain_input` but the input's `id` is `exponentlambdain_input_id`. Should we open a new issue for the active elements?

_Originally posted by @skulonen in https://github.com/apluslms/mooc-grader/issues/111#issuecomment-843192271_

Active element html is mostly created in the a-plus/exercise/static/chapter.js code.

**Comment by Mankro** [comment]
#964 fixed duplicate html `id="exercise"` in chapter and exercise pages. 

This issue (#823) contains the label `for` attribute bug in active elements.
The `<label for="">` attribute does not match the id of the input element and thus, clicking the label does not activate the text input like it should. The input's id attribute has a suffix `_id` which is missing in the label's `for` value.

In addition, please check if there are any obvious duplicate id attributes in active elements.
